.circleci/config.yml

```diff
@@ -20,19 +20,36 @@ jobs:
     working_directory: /go/src/github.com/kudobuilder/kudo
     steps:
       - checkout
+      - restore_cache:
+          keys:
+            - go-mod-v1-{{ checksum "go.sum" }}
 
       # specify any bash command here prefixed with `run: `
       - run: make check-formatting
```

Comment:
now that tests are working... `test` target should be part of the run requirement